ghost/i18n/locales/es/portal.json

```diff
@@ -6,63 +6,63 @@
     "{{amount}} off for first {{period}}.": "{{amount}} de descuento por los primeros {{period}}.",
     "{{amount}} off forever.": "{{amount}} de descuento para siempre.",
     "{{discount}}% discount": "{{discount}}% de descuento",
-    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ya no recibirá boletines de {{newsletterName}}.",
+    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ya no recibirá boletínes de {{newsletterName}}.",
     "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} ya no recibirá correos electrónicos cuando alguien responde a tus comentarios.",
     "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} ya no recibirá este boletín.",
     "{{trialDays}} days free": "{{trialDays}} días gratis",
-    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Se ha enviado un enlace de inicio de sesión a tu correo. Si no llega en 3 minutos, revisa tu carpeta de spam.",
+    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Se ha enviado un enlace de inicio de sesión a tu correo. Si no llega en 3 minutos, por favor revisa tu carpeta de spam.",
     "Account": "Cuenta",
     "Account settings": "Configuración de la cuenta",
     "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "Después de que finalice el período de prueba gratuito, se te cobrará el precio regular del nivel que hayas elegido. Siempre puedes cancelar antes de eso.",
     "Already a member?": "¿Ya eres miembro?",
     "An unexpected error occured. Please try again or <a>contact support</a> if the error persists.": "Un error inesperado ha ocurrido. Por favor intenta de nuevo o <a>contacta a soporte técnico</a> si el error persiste.",
     "Back": "Atrás",
-    "Back to Log in": "Volver al inicio de sesión",
+    "Back to Log in": "Atrás para Iniciar Sesión",
     "Billing info": "Información de pago",
     "Black Friday": "Black Friday",
-    "Cancel anytime.": "Cancelación en cualquier momento.",
+    "Cancel anytime.": "Cancela en cualquier momento.",
     "Cancel subscription": "Cancelar suscripción",
     "Cancellation reason": "Razón de cancelación",
     "Change": "Cambiar",
     "Check spam & promotions folders": "Comprueba la carpeta de spam y promociones",
-    "Check with your mail provider": "Comprueba con tu proveedor de correo",
+    "Check with your mail provider": "Comprueba con tu proveedor de correo electrónico",
     "Choose": "Elige",
     "Choose a different plan": "Elige un plan diferente",
-    "Choose your newsletters": "Elige tus boletines",
+    "Choose your newsletters": "Elige tus boletínes",
```

Comment:
is this a regional difference?  I'm getting i (not í) as the correct spelling for the plural...

Reply:
Wrong there too. Boletín is only with accent, not in plural, which is only boletines.